students/carlos_novoa/lesson02/series.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+"""
+Lesson2, Computing the Fibonacci and Lucas Series
+"""
+
+
+def fibonacci():
+    """Return a list containing a fibonacci series (starting at 0, 1)"""
+    return sum_series(False, 0, 1)
+
+
+def lucas():
+    """Return a list containing a lucas series (starting at 2, 1)"""
+    return sum_series(False, 2, 1)
```

Comment:
it looks like you would like the first argument to be optional, in this case you should make `n` keyword argument as well: `def sum_series(n=None, a=0, b=0)` then u could call it `rum_series(a=2, b=1)`